<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drehen_rechts">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>420</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>drehen rechts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bewegen_rechts">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>bewegen rechts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_naechster_stein">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>480</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>naechster Stein</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bewegen_hoch">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>bewegen hoch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_flip">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>flip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bewegen_runter">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>bewegen runter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drehen_links">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>drehen links</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bewegen_links">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>bewegen links</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_Spielfeld">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>401</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_Spielsteine">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_stein1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stein1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_stein2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stein2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_stein3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stein3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_stein4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stein4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_punkte">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Punkte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_eingaben">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Eingaben</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zeit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Restzeit</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_punkte">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_eingaben">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_zeit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;Logo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
